application/templates/cronjob.yaml

```diff
@@ -39,6 +39,9 @@ spec:
 {{ end }}
   jobTemplate:
     spec:
+      {{- with $job.startingDeadlineSeconds }}
```

Comment:
I've just seen that a lot of integers are included with the with keyword. However, this would skip all 0 values.  Let's keep it like this in this PR, I will handle the change in a new one.